modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/metal/js/components/RuleEditor/RuleEditor.es.js

```diff
@@ -567,6 +600,32 @@ class RuleEditor extends Component {
 		);
 	}
 
+	_fetchFunctionsURL() {
+		const {functionsURL} = this;
+
+		makeFetch(
+			{
+				method: 'GET',
+				url: functionsURL
+			}
+		).then(
+			responseData => {
+				if (!this.isDisposed()) {
+
+					this.setState(
+						{
+							calculatorOptions: responseData
+						}
+					);
+				}
+			}
+		).catch(
+			error => {
+				console.log(error);
```

Comment:
Remove console.log and try to create a handlerError method in a separate file (maybe an utils.js file... I think that we already have this kind of file created, but I'm not sure. Please double check it).